package.json

```diff
@@ -17,19 +17,22 @@
     "testEnvironment": "./test-environment.js"
   },
   "scripts": {
-    "clean": "rm -Rf node_modules ./modules/*/node_modules ./modules/*/dist",
+    "clean": "rm -Rf node_modules ./modules/*/node_modules ./modules/*/dist ./modules/*/.turbo",
     "lint": "tsdx lint modules/*/src",
     "build": "yarn turbo run build",
-    "test": "TESTING=true tsdx test --verbose --runInBand",
+    "test": "yarn workspaces run test",
     "docs:client": "jsdoc2md --files ./modules/client/src/*.ts ./modules/client/src/internal/client/*.ts --configure ./jsdoc2md.json > ./docs/client.md",
     "docs:addressList": "jsdoc2md --files ./modules/client/src/addressList/*.ts ./modules/client/src/addressList/internal/client/*.ts --configure ./jsdoc2md.json > ./docs/addressList.md",
     "docs:erc20": "jsdoc2md --files ./modules/client/src/erc20/*.ts ./modules/client/src/erc20/internal/client/*.ts --configure ./jsdoc2md.json > ./docs/erc20.md"
   },
   "devDependencies": {
     "@babel/preset-typescript": "^7.18.6",
+    "@ensdomains/ens-contracts": "^0.0.15",
+    "@types/node": "^18.11.9",
```

Comment:
Where are these types used? I would be very careful here, because the SDK has to work in browsers, Node is just an internal development tool we use with non-standard and many incompatible API's.

If this dependency gives us false "OK's" in terms of typing/linting, that's a problem.

Reply:
In the tests

Reply:
When you are developing (and even running the tests on the CI pipeline), the devDependencies are installed as well. You might get the wrong impression that some function will be fine, except that IRL it won't.

What is the NodeJS-only code that we need to run in the tests?

Reply:
we need it because we run ganache and need to deploy locally all the contracts
that is nodejs stuff

Reply:
Then, can we move this to the `package.json` of the client module only?